mnist/mnist_aws.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<p>Will this throw an exception if the user doesn't have credentials?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/739/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Actually sts won't fail here and it will use node role to make the call. I change to detail service we use in the notebook to check permissions</p><p> </p><p>It will throw failures like this. </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ClientError                               Traceback (most recent call last)
&lt;ipython-input-54-43d14d2701cc&gt; in &lt;module&gt;
      7 # Set REGION for s3 bucket and elastic contaienr registry
      8 AWS_REGION='us-west-2'
----&gt; 9 boto3.client('s3').list_buckets()

~/.local/lib/python3.6/site-packages/botocore/client.py in _api_call(self, *args, **kwargs)
    314                     "%s() only accepts keyword arguments." % py_operation_name)
    315             # The "self" in this scope is referring to the BaseClient.
--&gt; 316             return self._make_api_call(operation_name, kwargs)
    317 
    318         _api_call.__name__ = str(py_operation_name)

~/.local/lib/python3.6/site-packages/botocore/client.py in _make_api_call(self, operation_name, api_params)
    624             error_code = parsed_response.get("Error", {}).get("Code")
    625             error_class = self.exceptions.from_code(error_code)
--&gt; 626             raise error_class(parsed_response, operation_name)
    627         else:
    628             return parsed_response

ClientError: An error occurred (AccessDenied) when calling the ListBuckets operation: Access Denied
</pre>